---
title: Publications
description: |
  Publications from our scientists, fellows, associates, and interns
toc: false
sidebar: false
title-block-banner: true
---

## Preprints
Deng X,  **Frandsen P. B.**, **Dikow R. B.**, Favre, A., Shah, D. N., Shah, R. D. T., Schneider, J., Heckenhauer, J., Pauls, S. 2022. The impact of sequencing depth and relatedness of the reference genome in population genomic studies: a case study with two caddisfly species (Trichoptera, Rhyacophilidae, Himalopsyche). https://doi.org/10.22541/au.165633164.43557091/v1

Kawahara, A. Y., Storer, C. G., Markee, A., Heckenhauer, J., Powell, A., Plotkin, D., Hotaling, S., Cleland, T. P., **Dikow, R. B.**, Dikow, T., Kuranishi, R. B., Messcher, R., Pauls, S. U., Stewart, R. J., Tojo, K., **Frandsen, P. B.** 2022. Long-read HiFi Sequencing Correctly Assembles Repetitive heavy fibroin Silk Genes in New Moth and Caddisfly Genomes. bioRxiv 2022.06.01.494423; https://doi.org/10.1101/2022.06.01.494423

## Peer-reviewed publications
**Robillard, A.**, **Trizna, M. G.**, Ruiz-Tafur, K., Panduro, E. D., de Santana, C. D., **White, A. E.**, **Dikow, R. B.**, Deichmann, J. 2023. Application of a Deep Learning Image Classifier for Identification of Amazonian Fishes. Ecology and Evolution 13(5): e9987. https://doi.org/10.1002/ece3.9987

Madeline G. Bursell, **Rebecca B. Dikow**, Henrique V. Figueiró, Olga Dudchenko, Joseph P. Flanagan, Erez Lieberman Aiden, Benoit Goossens, Senthilvel K.S.S. Nathan, Warren E. Johnson, Klaus-Peter Koepfli, **Paul B. Frandsen**. 2022. Whole genome analysis of clouded leopard species reveals an ancient divergence and distinct demographic histories. iScience Volume 25, Issue 12. https://doi.org/10.1016/j.isci.2022.105647

Barbour, Nicole,  **Robillard, Alexander J.**,  Shillinger, George L.,  Lyubchich, Vyacheslav,  Secor, David H., Fagan, William F., and  Bailey, Helen.  2023. Clustering and Classification of Vertical Movement Profiles for Ecological Inference of Behavior. Ecosphere  14(1): e4384. https://doi.org/10.1002/ecs2.4384

Chavez, D. E., Gronau, I., Hains, T., **Dikow, R. B.**, **Frandsen, P. B.**, Johnson, W. E., Eizirik, E., Koepfli, K.-P., Wayne, R. K. 2022. Comparative genomics uncovers the evolutionary history, demography and molecular adaptations of South American canids. PNAS 119 (34) e2205986119. https://doi.org/10.1073/pnas.2205986119

Abreu, E. F., Pavan, S. E., **Tsuchiya, M. T. N.**, McLean, B. S., Wilson D. E., Percequillo, A. R., Maldonado, J. E. 2022.
Old specimens for old branches: Assessing effects of sample age in resolving a rapid Neotropical radiation of squirrels.
Molecular Phylogenetics and Evolution, https://doi.org/10.1016/j.ympev.2022.107576

Borowiec, M. L., **Dikow, R. B.**, **Frandsen, P. B.**, McKeeken, A., Valentini, G., & **White, A. E.** 2022. Deep learning as a tool for ecology and evolution. Methods in Ecology and Evolution https://doi.org/10.1111/2041-210X.13901

**Ekwealor, J. T. B.**, Benjamin, S. D., Jomsky, J. Z., Bowker, M. A., Stark, L. R., McLetchie, D. N., Mishler, B. D., Fisher, K. M. 2022. Genotypic confirmation of a biased phenotypic sex ratio in a dryland moss using restriction fragment length polymorphisms. Applications in Plant Sciences https://doi.org/10.1002/aps3.11467

Corbo, M., Damas, J., **Bursell, M. G.**, Lewin, H. A. 2022. Conservation of chromatin conformation in carnivores. Proceedings of the National Academy of Sciences, e2120555119. https://doi:10.1073/pnas.2120555119

Rana, S. K., **White, A. E.**, Price, T. D. 2022. Key roles for the freezing line and disturbance in driving the low plant species richness of temperate regions. Global Biogeography and Ecology  31(2);280-293 https://doi.org/10.1111/geb.13427

Besnier, C. and **Mattingly, W.**, 2021. Named-Entity Dataset for Medieval Latin, Middle High German and Old Norse. Journal of Open Humanities Data, 7, p.23. DOI: http://doi.org/10.5334/johd.36

**Arias, C. F.**, **Dikow, R. B.**,  McMillan, W. O., De Leon, L. F. 2021 De novo genome assembly of the electric fish Brachyhypopomus occidentalis (Hypopomidae, Gymnotiformes), Genome Biology and Evolution, evab223, https://doi.org/10.1093/gbe/evab223

Johnson, K. P., Burns, P. J., Stewart, J., Cook, T., Besnier, C. & **Mattingly, W. J. B.** 2021. "The Classical Language Toolkit: An NLP Framework for Pre-Modern Languages. Proceedings of the Joint Conference of the 59th Annual Meeting of the Association for Computational Linguistics and the 11th International Joint Conference on Natural Language Processing: System Demonstrations. https://aclanthology.org/2021.acl-demo.3.

Olsen, L. K.,  Heckenhauer, J.,  Sproul, J. S., **Dikow, R. B.**, Gonzalez, V. L., Kweskin, M. P., Taylor, A. M., Wilson, S. B., Stewart, R. J., Zhou, X., Holzenthal, R., Pauls, S. U., **Frandsen, P. B.** 2021.  Draft Genome Assemblies and Annotations of Agrypnia vestita Walker, and Hesperophylax magnus Banks Reveal Substantial Repetitive Element Expansion in Tube Case-making Caddisflies (Insecta: Trichoptera). Genome Biology and Evolution, https://doi.org/10.1093/gbe/evab013

**White, A. E.**, Dey, K. K., Stephens, M., Price, T. D. 2021. Dispersal syndromes drive the formation of biogeographical regions, illustrated by the case of Wallace's Line. Global Ecology and Biogeography https://doi.org/10.1111/geb.13250

Roa-Varón, A., **Dikow, R. B.**, Carnevale, G., Tornabene, L., Baldwin, C. C., Li, C., Hilton, E. J. 2020. Confronting Sources of Systematic Error to Resolve Historically Contentious Relationships: A Case Study Using Gadiform Fishes (Teleostei, Paracanthopterygii, Gadiformes), Systematic Biology, , syaa095, https://doi.org/10.1093/sysbio/syaa095

**Tsuchiya, M. T. N.**, **Dikow, R. B.**, Koepfli, K.-P., **Frandsen, P. B.**, Rockwood, L. L., Maldonado, J. E. 2020. Whole Genome Sequencing of Procyonids Reveals Distinct Demographic Histories in Kinkajou (Potos Flavus) and Northern Raccoon (Procyon Lotor), Genome Biology and Evolution, , evaa255, doi: https://doi.org/10.1093/gbe/evaa255

**Robillard, A.** 2020. Chelydra serpentina (Snapping Turtle). Predation. Herpetological Review 51(4):826-827. 

Feng, S., Stiller, J., Deng, Y. et al. 2020. Dense sampling of bird diversity increases power of comparative genomics. Nature 587, 252–257. doi: 10.1038/s41586-020-2873-9

**White, A. E.** 2020. Deep learning in deep time. Proceedings of the National Academy of Sciences. doi: 10.1073/pnas.2020870117

Legeai F., Santos B. F., Robin S., Bretaudeau A., **Dikow R. B.**, Lemaitre C., Jouan V., Ravallec M., Drezen J.-M., Tagu D., Baudat F., Gyapay G., Zhou X., Liu S., Webb B. A., Brady S. G., Volkoff A.-N. 2020. Genomic architecture of endogenous ichnoviruses reveals distinct evolutionary pathways leading to virus domestication in parasitic wasps. BMC Biology. doi: 10.1186/s12915-020-00822-3

Abreu-Jr Edson Fiedler de, Pavan Silvia E., **Tsuchiya Mirian T. N.**, Wilson Don E., Percequillo Alexandre R., Maldonado Jesús E. 2020. Spatiotemporal Diversification of Tree Squirrels: Is the South American Invasion and Speciation Really That Recent and Fast? Frontiers in Ecology and Evolution. doi: 10.3389/fevo.2020.00230   

**White, A. E.**, **Dikow, R. B.**, Baugh, M., Jenkins, A., **Frandsen, P. B.** 2020. Generating segmentation masks of herbarium specimens and a data set for training segmentation models using deep learning. Applications in Plant Sciences, invited article for the Special Issue: Machine Learning in Plant Biology. doi: 10.1002/aps3.11352

de Abreu-Jr, E.F., Pavan, S.E., **Tsuchiya, M.T.N.** *et al.* 2020. Museomics of tree squirrels: a dense taxon sampling of mitogenomes reveals hidden diversity, phenotypic convergence, and the need of a taxonomic overhaul. BMC Evol Biol 20, 77 (2020). doi: 10.1186/s12862-020-01639-y

Muga, K., **Robillard, A.**, Somaweera, R., 2020. Eretmochelys imbricata (Hawksbill Sea Turtle) Predation. Herpetological Review 51(2):313

**Tsuchiya, M. T. N.**, **Dikow, R. B.**, Cassin-Sackett, L. 2020. First genome sequence of the Gunnison’s prairie dog (Cynomys gunnisoni), a keystone species and player in the transmission of sylvatic plague. Genome Biology and Evolution 12: 5 618–625. doi: 10.1093/gbe/evaa069

Pearson K. D., Nelson G., Aronson M. F. J., Bonnet P., Brenskelle L., Davis C. C., Denny E. G., Ellwood E. R., Goëau H., Heberling J. M., Joly A., Lorieul T., Mazer S. J., Meineke E. K., Stucky B. J., Sweeney P., **White A. E.**, Soltis P. S. 2020. Machine Learning Using Digitized Herbarium Specimens to Advance Phenological Research, BioScience, biaa044. doi: 10.1093/biosci/biaa044

Jones, K. E., Fér, T., Schmickl, R., **Dikow, R. B.**, Funk, V. A., Herrando-Morraira, S., Johnston, P. R., Siniscalchi, C. M., Susanna, A., Slovák, M., Thapa, R., Watson, L., Mandel, J. R., 2019. An empirical assessment of a single family-wide target capture locus set at multiple evolutionary timescales within the largest family of flowering plants. Applications in Plant Sciences 7 (10):Article e11295. doi: 10.1002%2Faps3.11295

Edelman, N. B., **Frandsen, P. B.**, Miyagi, M., Clavijo, B. J., Davey, J., **Dikow, R. B.**, Accinelli, G. G., Van Belleghem, S., Patterson, N. J., Neafsey, D. E., Challis, R. J., Kumar, S., Moreira, G., Salazar, C., Chouteau, M., Counterman, B., Papa, R., Blaxter, M., Reed, R. D., Dasmahapatra, K., Kronforst, M., Joron, M., Jiggins, C. D., McMillan, W. O., Di-Palma, F., Blumberg, A. J., Wakeley, J., Jaffe, D., Mallet, J. 2019. Genomic architecture and introgression shape a butterfly radiation. Science 366 (6465), 594-599. doi: 10.1126/science.aaw2090

**Robillard, A.**, Robinson, S., Bastiaans, E., Vogler, D. 2019. Impacts of a highway on the population genetic structure of a threatened freshwater turtle (Glyptemys insculpta). Amphibian and Reptile Conservation. 13. 267-275. 

Schumm, M., Edie, S. M., Collins, K. S., Gómez-Bahamón, V., Supriya, K., **White, A. E.**, Price, T. D., Jablonski, D. 2019. Common latitudinal gradients in functional richness and functional evenness across marine and terrestrial systems. Proc. R. Soc. B 286: 20190745. doi: 10.1098/rspb.2019.0745

**White, A.E.**, Dey, K. K., Mohan, D. M., Stephens, M., Price, T. D. 2019. Regional influences on community structure   across the tropical-temperate divide. Nature Communications 10:2646.   doi: 10.1038/s41467-019-10253-6

Mandel, J. R., **Dikow, R. B.**, Siniscalchi, C. M., Thapa, R., Watson, L. E., Funk, V. A. 2019.  A fully resolved backbone phylogeny reveals numerous dispersals and explosive diversifications throughout thehistory  of  Asteraceae. Proceedings of the National Academy of Sciences 116 (28):14083-14088; doi:10.1073/pnas.1903871116.

Ohdera, A. H., Ames, C. L., **Dikow, R. B.**, Kayal, E., Chiodin, M., Busby, B., La, S., Pirro, S., Collins, A. G., Medina, M., Ryan, J. F. 2019. Box, stalked and upside-down? Draft genomes from diverse jellyfish (Cnidaria, Acraspeda) lineages: Alatina alata (Cubozoa), Calvadosia cruxmelitensis (Staurozoa), and Cassiopea xamachana (Scyphozoa). GigaScience 8(7). doi: 10.1093/gigascience/giz069

Cortes Rodrigues, N., Campana, M., Berry, L., Faegre, S., Derrickson, S., Ha, R., **Dikow, R. B.**, Rutz, C., Fleischer, R. 2019. Population genomics and structure of the critically endangered Mariana crow (Corvus kubaryi). Genes 10(3) doi: 10.3390/genes10030187.

Ren, Z., von Dohlen, C. D., Harris, A. J., **Dikow, R. B.**, Su, X., Wen, J. 2019.  Congruent phylogenetic relationships of Melaphidina aphids (Aphididae:  Eriosomatinae:  Fordini) accordingto nuclear and mitochondrial DNA data with taxonomic implications on generic limits. PLoS One doi: 10.1371/journal/pone.0213181

Hawkins, M. T. R., Culligan, R. C., **Dikow, R. B.**, Frasier, C.L., Lei, R., Louis Jr., E. E. 2018. Demographic history of the critically endangered greater bamboo lemur (Prolemur simus), evidence of population bottlenecks. BMC Genomics doi: 10.1186/s12864-018-4841-4.

Schuettpelz, E., **Frandsen, P. B.**, **Dikow, R. B.**, Brown, A., Orli, S., Peters, M., Metallo, A., Funk, V. A., Dorr, L. J. 2017. Applications of deep convolutional neural networks to digitized natural history collections. Biodiversity Data Journal. doi: 10.3897/BDJ.5.e21139 

Mandel, J. R., Barker, M. S., Bayer, R. J., **Dikow, R. B.**, Gao, T-G. Katy E. Jones, K. E., Keeley, S., Kilian, N., Ma, H, Siniscalchi, C. M., Susanna, A, Thapa, R., Watson, L., Funk, V. A. 2017. The Compositae Tree of Life in the age of phylogenomics. Journal of Systematics and Evolution. doi:10.1111/jse.12265 

Ren, Z. Harris, A. J., **Dikow, R. B.**, Ma, E., Zhong, Y., Wen, J. 2017. Another look at the phylogenetic relationships and intercontinental biogeography of eastern Asian & North American Rhus gall aphids (Hemiptera: Aphididae: Eriosomatinae): Evidence from mitogenome sequences via genome skimming. Molecular Phylogenetics and Evolution. doi: 10.1016/j.ympev.2017.05.017

Wen, J., Harris, A. J., Ickert-Bond, S. M., **Dikow, R. B.**, Wurdack, K, Zimmer, E. A. 2017. Developing integrative systematics in the informatics and genomic era, and calling for a global Biodiversity Cyberbank. Journal of Systematics and Evolution. doi:10.1111/jse.12270

**Dikow, R. B.**, **Frandsen, P. B.**, Turcatel, M., Dikow, T. 2017. Genomic and transcriptomic resources for assassin flies including the complete genome sequence of Proctacanthus coquilletti (Insecta: Diptera: Asilidae) and 16 representative transcriptomes. PeerJ. doi:10.7717/peerj.2951 
