<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.336">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="description" content="Publications from our scientists, fellows, associates, and interns">

<title>Smithsonian Data Science Lab - Publications</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./images/si_favicon.ico" rel="icon">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>


<link rel="stylesheet" href="dsl.css">
<meta property="og:title" content="Smithsonian Data Science Lab - Publications">
<meta property="og:description" content="Publications from our scientists, fellows, associates, and interns">
<meta property="og:site-name" content="Smithsonian Data Science Lab">
<meta name="twitter:title" content="Smithsonian Data Science Lab - Publications">
<meta name="twitter:description" content="Publications from our scientists, fellows, associates, and interns">
<meta name="twitter:site" content="@SIDataScience">
<meta name="twitter:card" content="summary">
</head>

<body class="nav-fixed fullcontent">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a href="./index.html" class="navbar-brand navbar-brand-logo">
    <img src="./images/dsl_logo.png" alt="Smithsonian Data Science Lab logo" class="navbar-logo">
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link" href="./people/index.html" rel="" target="">
 <span class="menu-text">Our team</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./research/index.html" rel="" target="">
 <span class="menu-text">Research</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./training/index.html" rel="" target="">
 <span class="menu-text">Training</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./news/index.html" rel="" target="">
 <span class="menu-text">News</span></a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-about-si-dsl" role="button" data-bs-toggle="dropdown" aria-expanded="false" rel="" target="">
 <span class="menu-text">About SI-DSL</span>
    </a>
    <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="nav-menu-about-si-dsl">    
        <li>
    <a class="dropdown-item" href="./about-dsl.html" rel="" target="">
 <span class="dropdown-text">Who we are</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./aivalues/index.html" rel="" target="">
 <span class="dropdown-text">AI Values Statement</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./publications.html" rel="" target="">
 <span class="dropdown-text">Publications</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./collaborators.html" rel="" target="">
 <span class="dropdown-text">Collaboration</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./contact.html" rel="" target="">
 <span class="dropdown-text">Contact us</span></a>
  </li>  
    </ul>
  </li>
</ul>
            <div class="quarto-navbar-tools">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<header id="title-block-header" class="quarto-title-block default page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <h1 class="title">Publications</h1>
                  <div>
        <div class="description">
          <p>Publications from our scientists, fellows, associates, and interns</p>
        </div>
      </div>
                </div>
  </div>
    
  
  <div class="quarto-title-meta">

      
    
      
    </div>
    
  
  </header><div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    
<!-- main -->
<main class="content quarto-banner-title-block" id="quarto-document-content">




<section id="preprints" class="level2">
<h2 class="anchored" data-anchor-id="preprints">Preprints</h2>
<p>Deng X, <strong>Frandsen P. B.</strong>, <strong>Dikow R. B.</strong>, Favre, A., Shah, D. N., Shah, R. D. T., Schneider, J., Heckenhauer, J., Pauls, S. 2022. The impact of sequencing depth and relatedness of the reference genome in population genomic studies: a case study with two caddisfly species (Trichoptera, Rhyacophilidae, Himalopsyche). https://doi.org/10.22541/au.165633164.43557091/v1</p>
<p>Kawahara, A. Y., Storer, C. G., Markee, A., Heckenhauer, J., Powell, A., Plotkin, D., Hotaling, S., Cleland, T. P., <strong>Dikow, R. B.</strong>, Dikow, T., Kuranishi, R. B., Messcher, R., Pauls, S. U., Stewart, R. J., Tojo, K., <strong>Frandsen, P. B.</strong> 2022. Long-read HiFi Sequencing Correctly Assembles Repetitive heavy fibroin Silk Genes in New Moth and Caddisfly Genomes. bioRxiv 2022.06.01.494423; https://doi.org/10.1101/2022.06.01.494423</p>
</section>
<section id="peer-reviewed-publications" class="level2">
<h2 class="anchored" data-anchor-id="peer-reviewed-publications">Peer-reviewed publications</h2>
<p>Madeline G. Bursell, <strong>Rebecca B. Dikow</strong>, Henrique V. Figueiró, Olga Dudchenko, Joseph P. Flanagan, Erez Lieberman Aiden, Benoit Goossens, Senthilvel K.S.S. Nathan, Warren E. Johnson, Klaus-Peter Koepfli, <strong>Paul B. Frandsen</strong>. 2022. Whole genome analysis of clouded leopard species reveals an ancient divergence and distinct demographic histories. iScience Volume 25, Issue 12. https://doi.org/10.1016/j.isci.2022.105647</p>
<p>Barbour, Nicole, <strong>Robillard, Alexander J.</strong>, Shillinger, George L., Lyubchich, Vyacheslav, Secor, David H., Fagan, William F., and Bailey, Helen. 2023. Clustering and Classification of Vertical Movement Profiles for Ecological Inference of Behavior. Ecosphere 14( 1): e4384. https://doi.org/10.1002/ecs2.4384</p>
<p>Chavez, D. E., Gronau, I., Hains, T., <strong>Dikow, R. B.</strong>, <strong>Frandsen, P. B.</strong>, Johnson, W. E., Eizirik, E., Koepfli, K.-P., Wayne, R. K. 2022. Comparative genomics uncovers the evolutionary history, demography and molecular adaptations of South American canids. PNAS 119 (34) e2205986119. https://doi.org/10.1073/pnas.2205986119</p>
<p>Abreu, E. F., Pavan, S. E., <strong>Tsuchiya, M. T. N.</strong>, McLean, B. S., Wilson D. E., Percequillo, A. R., Maldonado, J. E. 2022. Old specimens for old branches: Assessing effects of sample age in resolving a rapid Neotropical radiation of squirrels. Molecular Phylogenetics and Evolution, https://doi.org/10.1016/j.ympev.2022.107576</p>
<p>Borowiec, M. L., <strong>Dikow, R. B.</strong>, <strong>Frandsen, P. B.</strong>, McKeeken, A., Valentini, G., &amp; <strong>White, A. E.</strong> 2022. Deep learning as a tool for ecology and evolution. Methods in Ecology and Evolution https://doi.org/10.1111/2041-210X.13901</p>
<p><strong>Ekwealor, J. T. B.</strong>, Benjamin, S. D., Jomsky, J. Z., Bowker, M. A., Stark, L. R., McLetchie, D. N., Mishler, B. D., Fisher, K. M. 2022. Genotypic confirmation of a biased phenotypic sex ratio in a dryland moss using restriction fragment length polymorphisms. Applications in Plant Sciences https://doi.org/10.1002/aps3.11467</p>
<p>Corbo, M., Damas, J., <strong>Bursell, M. G.</strong>, Lewin, H. A. 2022. Conservation of chromatin conformation in carnivores. Proceedings of the National Academy of Sciences, e2120555119. https://doi:10.1073/pnas.2120555119</p>
<p>Rana, S. K., <strong>White, A. E.</strong>, Price, T. D. 2022. Key roles for the freezing line and disturbance in driving the low plant species richness of temperate regions. Global Biogeography and Ecology 31(2);280-293 https://doi.org/10.1111/geb.13427</p>
<p>Besnier, C. and <strong>Mattingly, W.</strong>, 2021. Named-Entity Dataset for Medieval Latin, Middle High German and Old Norse. Journal of Open Humanities Data, 7, p.23. DOI: http://doi.org/10.5334/johd.36</p>
<p><strong>Arias, C. F.</strong>, <strong>Dikow, R. B.</strong>, McMillan, W. O., De Leon, L. F. 2021 De novo genome assembly of the electric fish Brachyhypopomus occidentalis (Hypopomidae, Gymnotiformes), Genome Biology and Evolution, evab223, https://doi.org/10.1093/gbe/evab223</p>
<p>Johnson, K. P., Burns, P. J., Stewart, J., Cook, T., Besnier, C. &amp; <strong>Mattingly, W. J. B.</strong> 2021. “The Classical Language Toolkit: An NLP Framework for Pre-Modern Languages. Proceedings of the Joint Conference of the 59th Annual Meeting of the Association for Computational Linguistics and the 11th International Joint Conference on Natural Language Processing: System Demonstrations. https://aclanthology.org/2021.acl-demo.3.</p>
<p>Olsen, L. K., Heckenhauer, J., Sproul, J. S., <strong>Dikow, R. B.</strong>, Gonzalez, V. L., Kweskin, M. P., Taylor, A. M., Wilson, S. B., Stewart, R. J., Zhou, X., Holzenthal, R., Pauls, S. U., <strong>Frandsen, P. B.</strong> 2021. Draft Genome Assemblies and Annotations of Agrypnia vestita Walker, and Hesperophylax magnus Banks Reveal Substantial Repetitive Element Expansion in Tube Case-making Caddisflies (Insecta: Trichoptera). Genome Biology and Evolution, https://doi.org/10.1093/gbe/evab013</p>
<p><strong>White, A. E.</strong>, Dey, K. K., Stephens, M., Price, T. D. 2021. Dispersal syndromes drive the formation of biogeographical regions, illustrated by the case of Wallace’s Line. Global Ecology and Biogeography https://doi.org/10.1111/geb.13250</p>
<p>Roa-Varón, A., <strong>Dikow, R. B.</strong>, Carnevale, G., Tornabene, L., Baldwin, C. C., Li, C., Hilton, E. J. 2020. Confronting Sources of Systematic Error to Resolve Historically Contentious Relationships: A Case Study Using Gadiform Fishes (Teleostei, Paracanthopterygii, Gadiformes), Systematic Biology, , syaa095, https://doi.org/10.1093/sysbio/syaa095</p>
<p><strong>Tsuchiya, M. T. N.</strong>, <strong>Dikow, R. B.</strong>, Koepfli, K.-P., <strong>Frandsen, P. B.</strong>, Rockwood, L. L., Maldonado, J. E. 2020. Whole Genome Sequencing of Procyonids Reveals Distinct Demographic Histories in Kinkajou (Potos Flavus) and Northern Raccoon (Procyon Lotor), Genome Biology and Evolution, , evaa255, doi: https://doi.org/10.1093/gbe/evaa255</p>
<p><strong>Robillard, A.</strong> 2020. Chelydra serpentina (Snapping Turtle). Predation. Herpetological Review 51(4):826-827.</p>
<p>Feng, S., Stiller, J., Deng, Y. et al.&nbsp;2020. Dense sampling of bird diversity increases power of comparative genomics. Nature 587, 252–257. doi: 10.1038/s41586-020-2873-9</p>
<p><strong>White, A. E.</strong> 2020. Deep learning in deep time. Proceedings of the National Academy of Sciences. doi: 10.1073/pnas.2020870117</p>
<p>Legeai F., Santos B. F., Robin S., Bretaudeau A., <strong>Dikow R. B.</strong>, Lemaitre C., Jouan V., Ravallec M., Drezen J.-M., Tagu D., Baudat F., Gyapay G., Zhou X., Liu S., Webb B. A., Brady S. G., Volkoff A.-N. 2020. Genomic architecture of endogenous ichnoviruses reveals distinct evolutionary pathways leading to virus domestication in parasitic wasps. BMC Biology. doi: 10.1186/s12915-020-00822-3</p>
<p>Abreu-Jr Edson Fiedler de, Pavan Silvia E., <strong>Tsuchiya Mirian T. N.</strong>, Wilson Don E., Percequillo Alexandre R., Maldonado Jesús E. 2020. Spatiotemporal Diversification of Tree Squirrels: Is the South American Invasion and Speciation Really That Recent and Fast? Frontiers in Ecology and Evolution. doi: 10.3389/fevo.2020.00230</p>
<p><strong>White, A. E.</strong>, <strong>Dikow, R. B.</strong>, Baugh, M., Jenkins, A., <strong>Frandsen, P. B.</strong> 2020. Generating segmentation masks of herbarium specimens and a data set for training segmentation models using deep learning. Applications in Plant Sciences, invited article for the Special Issue: Machine Learning in Plant Biology. doi: 10.1002/aps3.11352</p>
<p>de Abreu-Jr, E.F., Pavan, S.E., <strong>Tsuchiya, M.T.N.</strong> <em>et al.</em> 2020. Museomics of tree squirrels: a dense taxon sampling of mitogenomes reveals hidden diversity, phenotypic convergence, and the need of a taxonomic overhaul. BMC Evol Biol 20, 77 (2020). doi: 10.1186/s12862-020-01639-y</p>
<p>Muga, K., <strong>Robillard, A.</strong>, Somaweera, R., 2020. Eretmochelys imbricata (Hawksbill Sea Turtle) Predation. Herpetological Review 51(2):313</p>
<p><strong>Tsuchiya, M. T. N.</strong>, <strong>Dikow, R. B.</strong>, Cassin-Sackett, L. 2020. First genome sequence of the Gunnison’s prairie dog (Cynomys gunnisoni), a keystone species and player in the transmission of sylvatic plague. Genome Biology and Evolution 12: 5 618–625. doi: 10.1093/gbe/evaa069</p>
<p>Pearson K. D., Nelson G., Aronson M. F. J., Bonnet P., Brenskelle L., Davis C. C., Denny E. G., Ellwood E. R., Goëau H., Heberling J. M., Joly A., Lorieul T., Mazer S. J., Meineke E. K., Stucky B. J., Sweeney P., <strong>White A. E.</strong>, Soltis P. S. 2020. Machine Learning Using Digitized Herbarium Specimens to Advance Phenological Research, BioScience, biaa044. doi: 10.1093/biosci/biaa044</p>
<p>Jones, K. E., Fér, T., Schmickl, R., <strong>Dikow, R. B.</strong>, Funk, V. A., Herrando-Morraira, S., Johnston, P. R., Siniscalchi, C. M., Susanna, A., Slovák, M., Thapa, R., Watson, L., Mandel, J. R., 2019. An empirical assessment of a single family-wide target capture locus set at multiple evolutionary timescales within the largest family of flowering plants. Applications in Plant Sciences 7 (10):Article e11295. doi: 10.1002%2Faps3.11295</p>
<p>Edelman, N. B., <strong>Frandsen, P. B.</strong>, Miyagi, M., Clavijo, B. J., Davey, J., <strong>Dikow, R. B.</strong>, Accinelli, G. G., Van Belleghem, S., Patterson, N. J., Neafsey, D. E., Challis, R. J., Kumar, S., Moreira, G., Salazar, C., Chouteau, M., Counterman, B., Papa, R., Blaxter, M., Reed, R. D., Dasmahapatra, K., Kronforst, M., Joron, M., Jiggins, C. D., McMillan, W. O., Di-Palma, F., Blumberg, A. J., Wakeley, J., Jaffe, D., Mallet, J. 2019. Genomic architecture and introgression shape a butterfly radiation. Science 366 (6465), 594-599. doi: 10.1126/science.aaw2090</p>
<p><strong>Robillard, A.</strong>, Robinson, S., Bastiaans, E., Vogler, D. 2019. Impacts of a highway on the population genetic structure of a threatened freshwater turtle (Glyptemys insculpta). Amphibian and Reptile Conservation. 13. 267-275.</p>
<p>Schumm, M., Edie, S. M., Collins, K. S., Gómez-Bahamón, V., Supriya, K., <strong>White, A. E.</strong>, Price, T. D., Jablonski, D. 2019. Common latitudinal gradients in functional richness and functional evenness across marine and terrestrial systems. Proc. R. Soc. B 286: 20190745. doi: 10.1098/rspb.2019.0745</p>
<p><strong>White, A.E.</strong>, Dey, K. K., Mohan, D. M., Stephens, M., Price, T. D. 2019. Regional influences on community structure across the tropical-temperate divide. Nature Communications 10:2646. doi: 10.1038/s41467-019-10253-6</p>
<p>Mandel, J. R., <strong>Dikow, R. B.</strong>, Siniscalchi, C. M., Thapa, R., Watson, L. E., Funk, V. A. 2019. A fully resolved backbone phylogeny reveals numerous dispersals and explosive diversifications throughout thehistory of Asteraceae. Proceedings of the National Academy of Sciences 116 (28):14083-14088; doi:10.1073/pnas.1903871116.</p>
<p>Ohdera, A. H., Ames, C. L., <strong>Dikow, R. B.</strong>, Kayal, E., Chiodin, M., Busby, B., La, S., Pirro, S., Collins, A. G., Medina, M., Ryan, J. F. 2019. Box, stalked and upside-down? Draft genomes from diverse jellyfish (Cnidaria, Acraspeda) lineages: Alatina alata (Cubozoa), Calvadosia cruxmelitensis (Staurozoa), and Cassiopea xamachana (Scyphozoa). GigaScience 8(7). doi: 10.1093/gigascience/giz069</p>
<p>Cortes Rodrigues, N., Campana, M., Berry, L., Faegre, S., Derrickson, S., Ha, R., <strong>Dikow, R. B.</strong>, Rutz, C., Fleischer, R. 2019. Population genomics and structure of the critically endangered Mariana crow (Corvus kubaryi). Genes 10(3) doi: 10.3390/genes10030187.</p>
<p>Ren, Z., von Dohlen, C. D., Harris, A. J., <strong>Dikow, R. B.</strong>, Su, X., Wen, J. 2019. Congruent phylogenetic relationships of Melaphidina aphids (Aphididae: Eriosomatinae: Fordini) accordingto nuclear and mitochondrial DNA data with taxonomic implications on generic limits. PLoS One doi: 10.1371/journal/pone.0213181</p>
<p>Hawkins, M. T. R., Culligan, R. C., <strong>Dikow, R. B.</strong>, Frasier, C.L., Lei, R., Louis Jr., E. E. 2018. Demographic history of the critically endangered greater bamboo lemur (Prolemur simus), evidence of population bottlenecks. BMC Genomics doi: 10.1186/s12864-018-4841-4.</p>
<p>Schuettpelz, E., <strong>Frandsen, P. B.</strong>, <strong>Dikow, R. B.</strong>, Brown, A., Orli, S., Peters, M., Metallo, A., Funk, V. A., Dorr, L. J. 2017. Applications of deep convolutional neural networks to digitized natural history collections. Biodiversity Data Journal. doi: 10.3897/BDJ.5.e21139</p>
<p>Mandel, J. R., Barker, M. S., Bayer, R. J., <strong>Dikow, R. B.</strong>, Gao, T-G. Katy E. Jones, K. E., Keeley, S., Kilian, N., Ma, H, Siniscalchi, C. M., Susanna, A, Thapa, R., Watson, L., Funk, V. A. 2017. The Compositae Tree of Life in the age of phylogenomics. Journal of Systematics and Evolution. doi:10.1111/jse.12265</p>
<p>Ren, Z. Harris, A. J., <strong>Dikow, R. B.</strong>, Ma, E., Zhong, Y., Wen, J. 2017. Another look at the phylogenetic relationships and intercontinental biogeography of eastern Asian &amp; North American Rhus gall aphids (Hemiptera: Aphididae: Eriosomatinae): Evidence from mitogenome sequences via genome skimming. Molecular Phylogenetics and Evolution. doi: 10.1016/j.ympev.2017.05.017</p>
<p>Wen, J., Harris, A. J., Ickert-Bond, S. M., <strong>Dikow, R. B.</strong>, Wurdack, K, Zimmer, E. A. 2017. Developing integrative systematics in the informatics and genomic era, and calling for a global Biodiversity Cyberbank. Journal of Systematics and Evolution. doi:10.1111/jse.12270</p>
<p><strong>Dikow, R. B.</strong>, <strong>Frandsen, P. B.</strong>, Turcatel, M., Dikow, T. 2017. Genomic and transcriptomic resources for assassin flies including the complete genome sequence of Proctacanthus coquilletti (Insecta: Diptera: Asilidae) and 16 representative transcriptomes. PeerJ. doi:10.7717/peerj.2951</p>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">Copyright © 2023 <a href="https://www.si.edu/">Smithsonian</a></div>   
    <div class="nav-footer-center">
      <ul class="footer-items list-unstyled">
    <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/sidatasciencelab">
      <i class="bi bi-github" role="img" aria-label="GitHub">
</i> 
    </a>
  </li>  
    <li class="nav-item compact">
    <a class="nav-link" href="https://twitter.com/SIDataScience">
      <i class="bi bi-twitter" role="img" aria-label="Twitter">
</i> 
    </a>
  </li>  
</ul>
    <div class="toc-actions"><div><i class="bi bi-github"></i></div><div class="action-links"><p><a href="https://github.com/sidatasciencelab/sidatasciencelab.github.io/edit/main/publications.qmd" class="toc-action">Edit this page</a></p><p><a href="https://github.com/sidatasciencelab/sidatasciencelab.github.io/issues/new" class="toc-action">Report an issue</a></p></div></div></div>
    <div class="nav-footer-right"><a href="https://www.si.edu/Termsofuse">Terms of Use</a></div>
  </div>
</footer>



</body></html>